### ISSUE TITLE
Error installing Node.ACS

### ISSUE THREAD

**Issue by kipstafoo** [npm/npm#6736]
Running this on OSX 10.9.5. I've tried to delete node, clean the cache, pretty much everything I can think of. Error remains the same.

```
new-host:local kipsta$ sudo npm install -g acs
npm http GET https://registry.npmjs.org/acs
npm http 200 https://registry.npmjs.org/acs
npm http GET https://registry.npmjs.org/acs/-/acs-1.0.20.tgz
npm http 200 https://registry.npmjs.org/acs/-/acs-1.0.20.tgz
npm ERR! tar pack Error reading /Users/kipsta/tmp/npm-693/1416395496292-0.7166676917113364/package
npm ERR! TypeError: Cannot call method 'filter' of undefined
npm ERR!     at Packer.IgnoreReader.addIgnoreRules (/usr/local/Cellar/node/0.8.20/lib/node_modules/npm/node_modules/fstream-npm/node_modules/fstream-ignore/ignore.js:148:13)
npm ERR!     at Packer.IgnoreReader.addIgnoreFile (/usr/local/Cellar/node/0.8.20/lib/node_modules/npm/node_modules/fstream-npm/node_modules/fstream-ignore/ignore.js:133:10)
npm ERR!     at fs.readFile (fs.js:176:14)
npm ERR!     at Object.oncomplete (fs.js:297:15)
npm ERR! If you need help, you may report this log at:
npm ERR!     <http://github.com/isaacs/npm/issues>
npm ERR! or email it to:
npm ERR!     <npm-@googlegroups.com>
```

**Comment by smikes** [comment]
Is this still a problem for you?

Could you please upload your npm-debug.log as a  gist <http://gist.github.com>? If possible, run npm with at least one `-d` flag to increase the logging level.

There have been a lot of improvements to npm -- especially around conflicts and race conditions during install -- since 1.2.11. Can you try updating your npm installation?

To update npm, run `npm -g install npm@latest`

We are trying to clean up older npm issues, so if we don't hear back from you within a week, we will close this issue. (Don't worry -- you can always come back again and open a new issue!)

Thanks!

**Comment by othiym23** [comment]
Closing as resolved -- `npm@1.2.11` is way old and unsupported; upgrade and this problem should no longer exist.
